config/locales/en/javascript/course/assessment/submission/submissions.yml

```diff
@@ -0,0 +1,12 @@
+en:
+  javascript:
+    course:
+      assessment:
+        submission:
+          submissions:
+            exam:
+              notice: >
+                You are entering an exam.
+
+                Please do not sign out or close the browser, otherwise you will have trouble
```

Comment:
`Please do not sign out or close the browser, otherwise you will require the course staff's assistance to continue with the exam.`